### PR TITLE
mkcloud, qa_crowbarsetup: show info about jenkins job in /etc/motd

### PR DESCRIPTION
**Pull request by jdsn**
to make it easy to find the job that created a cloud the /etc/motd shows the relevant jenkins information incl. the job URL

### CODE REVIEW

**Comment by bmwiedemann** [comment]
this would only address half of issue #759

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -1134,8 +1134,17 @@ function onadmin_add_cloud_repo()
         done
     fi
 
-    echo -n "This cloud was installed on `cat ~/cloud` from: " | \
-        cat - /etc/cloudversion >> /etc/motd
+    cat >>/etc/motd <<EOMOTD
+This cloud was installed on `cat ~/cloud` from: `cat /etc/cloudversion`
+`if [[ $JENKINS_BUILD_URL ]] ; then
```
Comment (aspiers) [inline scripts/qa_crowbarsetup.sh]: Ugh no!  Don't nest complex code inside `cat`.  Instead:

    (
        echo "This cloud was installed on `cat ~/cloud` from: `cat /etc/cloudversion`"
        if [[ $JENKINS_BUILD_URL ]] ; then
            echo "Installed via Jenkins"
            echo "  created by the job:    $JENKINS_BUILD_URL"
            echo "  on the Jenkins worker: $JENKINS_NODE_NAME"
            echo "  using executor number: $JENKINS_EXECUTOR_NUMBER"
            echo "  using workspace path:  $JENKINS_WORKSPACE"
        fi
    ) >> /etc/motd

**Comment by aspiers** [comment]
@bmwiedemann That's exactly what we just discussed on IRC :)

**Comment by jdsn** [comment]
> this would only address half of issue #759

issue #759 and PR #760 were created in parallel, so the chance to cover 100% was very small anyway :)

**Comment by bmwiedemann** [comment]
:+1:

**Comment by aplanas** [comment]
+1

**Comment by aspiers** [comment]
+1, thanks!

**Comment by aspiers** [comment]
@jdsn Looks like you manually aborted [the gating build](https://ci.suse.de/job/openstack-mkcloud/14703/consoleFull) - presumably because a full gating run is not necessary?  I guess you connected to `vd1` and verified the contents of `/etc/motd`?

**Comment by jdsn** [comment]
> @jdsn Looks like you manually aborted the gating build - presumably because a full gating run is not necessary? I guess you connected to vd1 and verified the contents of /etc/motd?

Nope, because it was blocking for too long. Maybe the github API was down at that moment.

**Comment by jdsn** [comment]
Manually verified that it works (login on admin node).

**Pull request merged**